Python/generated_cases.c.h

```diff
@@ -6009,6 +6009,7 @@
                     stack_pointer = _PyFrame_GetStackPointer(frame);
                 }
             }
+            stack_pointer[-1] = iter;
```

Comment:
Not something to be fixed here, but worth noting that this is unnessesary (the only paths that don't store `iter` set `iter = iterable`, which leaves the stack unchanged). Could be pretty difficult for the code generator to reason about though, I don't know...